### ISSUE TITLE
Accessibility issue "in Forms → Autocomplete"

### ISSUE THREAD

**Issue by roelvangils** [digipolisantwerp/acpaas-ui_angular#223]
## Describe the bug

The dropdown looks and behaves like a list box (visually) but some roles are missing. In accordance with WCAG 2.1, this does not satisfy [1.3.1 Info and Relationships](https://wcag.be/131#info-and-relationships). In its current implementation, the list box is not usable by users of AT.

## How to fix

Apply at least these attributes:

- List items need `role="option"`
- Turn the `<ul>` into a listbox with `role="listbox"`

## Additional information

- See [ARIA: listbox role - Accessibility | MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role) for more information.
- [Listbox Example with Grouped Options | WAI-ARIA Authoring Practices 1.2](https://w3c.github.io/aria-practices/examples/listbox/listbox-grouped.html)
